colandr/api/resources/fulltext_uploads.py

```diff
@@ -2,23 +2,22 @@
 import os
 import subprocess
 
+import ftfy
 from flask import current_app, g
 from flask_restplus import Resource
-from werkzeug.utils import secure_filename
-
 from marshmallow import fields as ma_fields
 from marshmallow.validate import Range
 from webargs.flaskparser import use_kwargs
-
-from textacy.preprocess import fix_bad_unicode
+from werkzeug.utils import secure_filename
 
 from colandr import api_
+
 from ...lib import constants
```

Comment:
Just a stylistic question, but should we move to Absolute imports?

Reply:
tbh i don't have a strong preference! (this is rare for me when it comes to python style, haha)

at some point i'm going to blacken this entire code base, and maybe then i'll revisit our import conventions